src/server/pfs/pretty/pretty.go

```diff
@@ -19,7 +19,7 @@ const (
 	// RepoAuthHeader is the header for repos with auth information attached.
 	RepoAuthHeader = "NAME\tCREATED\tSIZE (MASTER)\tACCESS LEVEL\t\n"
 	// CommitHeader is the header for commits.
-	CommitHeader = "REPO\tBRANCH\tCOMMIT\tFINISHED\tSIZE\tPROGRESS\tDESCRIPTION\n"
+	CommitHeader = "REPO\tBRANCH\tCOMMIT\tFINISHED\tSIZE\tDESCRIPTION\n"
```

Comment:
Is it going to be possible to get this progress information from somewhere? It seems like it would just be the sums of the status of the commits within a particular job but I might be oversimplifying.

Reply:
Yes - the CLI updates will happen in a separate PR, though.